rustsec/src/database.rs

```diff
@@ -15,10 +15,13 @@ use crate::{
     vulnerability::Vulnerability,
     Lockfile,
 };
-use std::path::Path;
+use std::path::{Path, PathBuf};
 
-#[cfg(feature = "git")]
-use crate::repository::git;
+/// Directory under `~/.cargo` where the advisory-db repo will be kept
+const ADVISORY_DB_DIRECTORY: &str = "advisory-db";
+
+/// The default URL from which the DB is downloaded
+const DEFAULT_DB_URL: &str = "https://github.com/zip-rs/zip/archive/refs/heads/master.zip";
```

Comment:
This seems bogus in multiple ways... just for testing I guess?

FWIW the tarball seems a lot smaller:

- 463 kB: https://github.com/rustsec/advisory-db/archive/refs/heads/main.zip
- 180 kB: https://github.com/rustsec/advisory-db/tarball/main

Reply:
Anything else I should fix here other than switching from .zip to .tar.gz?

Reply:
Looking good so far otherwise. Hopefully etags work for those resources.

Reply:
Well they do provide an etag in the response, so the worst case is we don't rely on the 304 and instead check the etag in the response headers and stop there. So no worries there.